packages/teuchos/core/test/UnitTest/CMakeLists.txt

```diff
@@ -38,13 +39,13 @@ TRIBITS_ADD_EXECUTABLE_AND_TEST(
   )
 
 IF (${PARENT_PACKAGE_NAME}_ENABLE_COMPLEX AND ${PARENT_PACKAGE_NAME}_ENABLE_FLOAT)
-  SET(PASS_STRING "Summary: total = 18, run = 17, passed = 17, failed = 0")
+  SET(PASS_STRING "Summary: total = 19, run = 18, passed = 18, failed = 0")
```

Comment:
You would have to update this every time you add a test; that would require rerunning CMake.  Why not just search for "failed = 0"?

Reply:
I think I added that because I was paranoid that someone would accidentally drop out all (or some) of the unit tests and it would trivially pass.

Reply:
That's reasonable.  We could do some CMake math on "run" and "passed" too....